### PR TITLE
Hide old slots on Booking modal

### PR DESCRIPTION
**Pull request by rafcontreras**
If approved, this PR will:

Hide slots in the past in booking modal

### CODE REVIEW

File: src/calendar/BookingLocation.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /* eslint-disable react/jsx-no-target-blank */
 import { Button } from "baseui/button";
+import isToday from "date-fns/isToday";
+import isAfter from "date-fns/isAfter";
```
Comment (olafwrieden) [inline src/calendar/BookingLocation.tsx:4]: Can these two be imported in one line like `import { isToday, isAfter } from 'date-fns`?

**Review by olafwrieden — CHANGES_REQUESTED** [review]
Minor change on the import (if possible) but code looks good to me, I haven't tested or checked for edge cases / times.

**Comment by noway** [comment]
I honestly thought we did that already?
@wseagar Is it a regression introduced with the real time modal feature? 

I'll give proper review tomorrow (today)

File: src/calendar/BookingLocation.tsx (hunk 1 above)
Comment (rafcontreras) [reply src/calendar/BookingLocation.tsx]: Changes done

**Comment by wseagar** [comment]
@noway yeah so the real time modal feature is not filtering slots that are before the current time. 

I think we do have this code somewhere for the data that's being retrieved from git. This data is retrieved at a different time and is stored in local state so I wouldn't worry too much about sharing the code.

Looks good to me.

**Comment by wseagar** [comment]
I might be going crazy I can't find any code which is filtering old slots anywhere. Maybe we never added it? Anyway this code will filter old slots for stale or live data so it is the correct solution.

**Comment by noway** [comment]
@olafwrieden your requested change was made, can you approve now

**Review by olafwrieden — APPROVED** [review]

**Pull request merged**